cmd/influx_inspect/export.go

```diff
@@ -17,19 +19,21 @@ import (
 )
 
 type cmdExport struct {
-	path            string
+	dataDir         string
+	walDir          string
 	out             string
 	db              string
 	retentionPolicy string
 	startTime       int64
 	endTime         int64
 	compress        bool
 
-	ext   string
-	files map[string][]string
+	manifest map[string]struct{}
+	tsmFiles map[string][]string
+	walFiles map[string][]string
 }
 
-func newCmdExport(path, out, db, retentionPolicy, startTime, endTime string, compress bool) (*cmdExport, error) {
+func newCmdExport(dataDir, walDir, out, db, retentionPolicy, startTime, endTime string, compress bool) (*cmdExport, error) {
```

Comment:
This is looking like it would benefit from a configuration struct instead of eight arguments.